File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int t) {
        Map<Integer,Integer>mp = new HashMap<>();
        for(int i=0;i<nums.length;i++){
            int v=t-nums[i];
            if(mp.containsKey(v)) return new int[]{mp.get(v),i};
            mp.put(nums[i],i);
        }
        return new int[]{};
    }
}

File: Data Structures & Algorithms/remove-element/submission-1.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int c=0,v=0;
        for( int i:nums){
            if(i!=val){
                nums[v++]=i;
                c++;
            }
        }
        return c;
    }
};